components/Header.js

```diff
@@ -15,6 +15,6 @@ export function Header(props) {
 
 const styles = StyleSheet.create({
   header: {
-    flex: 0.2
+    height: 70
```

Comment:
Whoops, I should've commented that here and not in the snapshot file. I'm a little hesitant to use the hard coded styling due to it being less accurate formatting across device sizes. However, this is totally sufficient for now and something we can look into later when polishing.

Reply:
```import { Dimensions, Slider, StyleSheet, Text, View } from 'react-native';

const height = Dimensions.get('window').height;

.
.
.

const styles = StyleSheet.create({
  header: {
    flex: 0.2,
    height: 0.2 * height
}
})
 @vrandall66
vrandall66 now Member